android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -198,7 +199,13 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
             if (view.playerView != null) {
                 view.player = player
             } else {
-                val playerView = PlayerView(context, player)
+                // PlayerView has to be initialized with Activity context
+                val currentActivity = context.currentActivity
+                if (currentActivity == null) {
+                    Log.e(MODULE_NAME, "Cannot create a PlayerView, because no activity is attached.")
+                    return@post
+                }
+                val playerView = PlayerView(currentActivity, player)
```

Comment:
This issue doesn't seem React native specific; could `PlayerView` constructor throw (or at the very least log an error) when it receive an application context instead of an activity one?

Reply:
Let's discuss outside this PR 😊